### ISSUE TITLE
Hard to put dataset in the right place for just a test.

### ISSUE THREAD

**Issue by aragorn1025** [Vandermode/ERRNet#8]
Is there any hint for put dataset in the right place for at least a test?

**Comment by MenSanYan** [comment]
@Vandermode @aragorn1025 What should I do for testing only one image? Can you provide an example for inference ? Anticipate for your reply.

**Comment by aditya98ak** [comment]
Hi @MenSanYan, I was also trying to test for one single image.
I did the following to get the results.
-  Created a folder, placed my testing image inside that directory
- Commented all code related to the creation of dataset and data loader
- Created my dataset for example, test_ref_dataset = datasets.RealDataset('/home/aditya/project/ERRNet/test_ref/')
- Created a dataloader for the same 
- test_ref_dataloader = datasets.DataLoader(
    test_ref_dataset, batch_size=8, shuffle=False,
    num_workers=opt.nThreads, pin_memory=True)
- res = engine.test(test_ref_dataloader, savedir=result_dir)
After this, I was able to get for a single image in the results directory.

**Comment by MohammadSamaraChowagiken** [comment]
Can we use this network for commercial purpose, as a part of an image processing processes.

**Comment by innat** [comment]
To inference on raw samples. In `test_errnet.py`, right after line 17, commented rest of the part. And put below code 

```
engine = Engine(opt)
test_ref_dataset = datasets.RealDataset('/content/my_img/') # where you save your image

test_ref_dataloader = datasets.DataLoader(
                    test_ref_dataset, batch_size=8, shuffle=False,
                    num_workers=opt.nThreads, pin_memory=True)

res = engine.test(test_ref_dataloader, savedir="/content/") # where result would be saved 
```

**Comment by WGHao** [comment]
Can i use the code without ubuntu?

**Comment by keweijian** [comment]
Hello, can you publish the misaligned dataset used in the project?

**Comment by 77453460** [comment]
> To inference on raw samples. In `test_errnet.py`, right after line 17, commented rest of the part. And put below code
> 
> ```
> engine = Engine(opt)
> test_ref_dataset = datasets.RealDataset('/content/my_img/') # where you save your image
> 
> test_ref_dataloader = datasets.DataLoader(
>                     test_ref_dataset, batch_size=8, shuffle=False,
>                     num_workers=opt.nThreads, pin_memory=True)
> 
> res = engine.test(test_ref_dataloader, savedir="/content/") # where result would be saved 
> ```

为什么我按照这样推理出来是一张纯黑的图
